cassandra-spark/src/main/scala/geotrellis/spark/store/cassandra/CassandraLayerWriter.scala

```diff
@@ -22,14 +22,17 @@ import geotrellis.store.avro._
 import geotrellis.store.avro.codecs._
 import geotrellis.store.cassandra._
 import geotrellis.store.index._
-import geotrellis.layer.merge.Mergable
 import geotrellis.spark.store._
 import geotrellis.spark.merge._
 import geotrellis.util._
 
 import com.typesafe.scalalogging.LazyLogging
+
 import org.apache.spark.rdd.RDD
-import io.circe._
+
+import _root_.io.circe._
```

Comment:
why is it a `_root_` now?

Reply:
Consistency throughout the library; certainly not necessity

Reply:
mb lets use _root_ where is necessary? originally I tired to use it only in places were packages were overlapped.

Reply:
Looks good otherwise.